src/sharing/scss/_app.scss

```
@@ -0,0 +1,22 @@
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 500;
+  line-height: 1.2;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Can we include a Google font, Roboto, to stay close to what it is now?

```suggestion
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```